src/utils/components/DiskModal/DiskFormFields/AdvancedSettings.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Dispatch, FC } from 'react';
+
+import HelpTextIcon from '@kubevirt-utils/components/HelpTextIcon/HelpTextIcon';
+import { FEATURE_HCO_PERSISTENT_RESERVATION } from '@kubevirt-utils/hooks/useFeatures/constants';
+import { useFeatures } from '@kubevirt-utils/hooks/useFeatures/useFeatures';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { diskTypes } from '@kubevirt-utils/resources/vm/utils/disk/constants';
+import { Checkbox, ExpandableSection, Split, Stack, StackItem } from '@patternfly/react-core';
+
+import { diskReducerActions, DiskReducerActionType } from '../state/actions';
+import { DiskFormState } from '../state/initialState';
+
+type AdvancedSettingsProps = {
+  diskState: DiskFormState;
+  dispatchDiskState: Dispatch<DiskReducerActionType>;
+};
+
+const AdvancedSettings: FC<AdvancedSettingsProps> = ({ diskState, dispatchDiskState }) => {
+  const { t } = useKubevirtTranslation();
+
+  const { featureEnabled } = useFeatures(FEATURE_HCO_PERSISTENT_RESERVATION);
+
+  const isLunType = diskState.diskType === diskTypes.lun;
+  return (
+    <ExpandableSection toggleText={t('Advanced Settings')}>
```

Comment:
I think we prefer sentence case, so maybe 'Advanced settings' would be more aligned with the rest of the UI.